packages/bpk-component-datatable/README.md

```diff
@@ -55,6 +131,9 @@ in addition to the following:
 | headerHeight           | number                      | false    | 60                   |
 | rowHeight              | number                      | false    | 60                   |
 | defaultColumnSortIndex | number                      | false    | 0                    |
+| sort                   | func                        | false    | null                 |
+| sortBy                 | string                      | false    | null                 |
+| sortDirection          | oneOf('ASC', 'DESC')        | false    | null                 |
```

Comment:
Its mentioned sortDirection its type is `oneOf('ASC', 'DESC')` but the prop type defined in the implementation just accepts a string. 

So probably if we are restricting it to be one of these two values we need to make the prop explcit in the implementation

so in the `propTypes` declaration in the main file set it to 

`sortDirection: PropTypes.oneOf('ASC', 'DESC')`

Left the suggestion below